passes/techmap/extract_counter.cc

```diff
@@ -372,13 +384,8 @@ void greenpak4_counters_worker(
 		return;
 	}
 
-	//Figure out the final cell type based on the counter size
-	string celltype = "\\GP_COUNT8";
-	if(extract.width > 8)
-		celltype = "\\GP_COUNT14";
-
 	//Get new cell name
-	string countname = string("$auto$GP_COUNTx$") + log_id(extract.rwire->name.str());
+	string countname = string("$auto$COUNTx$") + log_id(extract.rwire->name.str());
```

Comment:
`$auto` is the prefix used by IDs generated by NEW_ID. Simply using for example `$COUNTx$...` would be sufficient here.